packages/block-editor/src/components/block-settings-menu/block-html-convert-button.js

```diff
@@ -1,7 +1,7 @@
 /**
  * WordPress dependencies
  */
-import { rawHandler, getBlockContent } from '@wordpress/blocks';
+import { rawHandler, getBlockInnerHTML } from '@wordpress/blocks';
```

Comment:
It would be a breaking change and could negatively impact the broader WordPress ecosystem.